dev-scripts/deploy_custom_rh-che.sh

```diff
@@ -391,8 +398,8 @@ CHE_CONFIG_YAML=$(echo "$CHE_CONFIG_YAML" | \
 
 if [ "$RH_CHE_USE_CUSTOM_REGISTRIES" == "true" ]; then
   CHE_CONFIG_YAML=$(echo "$CHE_CONFIG_YAML" | \
-                    yq ".\"data\".\"CHE_WORKSPACE_PLUGIN__REGISTRY__URL\" = \"http://che-plugin-registry-${RH_CHE_PROJECT_NAMESPACE}.${RH_CHE_OPENSHIFT_URL_HOSTNAME}/v3\" |
-                        .\"data\".\"CHE_WORKSPACE_DEVFILE__REGISTRY__URL\" = \"http://che-devfile-registry-${RH_CHE_PROJECT_NAMESPACE}.${RH_CHE_OPENSHIFT_URL_HOSTNAME}/\" ")
+                    yq ".\"data\".\"CHE_WORKSPACE_PLUGIN__REGISTRY__URL\" = \"http$SECURE://che-plugin-registry-${RH_CHE_PROJECT_NAMESPACE}.apps.che-dev.x6e0.p1.openshiftapps.com/v3\" |
+                        .\"data\".\"CHE_WORKSPACE_DEVFILE__REGISTRY__URL\" = \"http$SECURE://che-devfile-registry-${RH_CHE_PROJECT_NAMESPACE}.apps.che-dev.x6e0.p1.openshiftapps.com/\" ")
```

Comment:
Could you please explain why the previous variable RH_CHE_OPENSHIFT_URL_HOSTNAME is not suitable here?